lib/milc_interface.cpp

```diff
@@ -1136,7 +1136,7 @@ void qudaEigCGInvert(int external_precision,
 {
   static const QudaVerbosity verbosity = QUDA_VERBOSE; //getVerbosity();
   qudamilc_called<true>(__func__, verbosity);
-
+#if 0
```

Comment:
@maddyscientist  yes, this must be updated (did not mention this point explicitly in the TODO ). Deflated solvers are set up via unified interface structure QudaEigParam, and are running in invertQuda , same as multigrid. Also deflation_test.cpp must be removed (another point I forgot to check) which is replaced by deflated_invert_test.cpp .